tests/draft2019-09/unevaluatedItems.json

```diff
@@ -413,5 +413,25 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "unevaluatedItems can't see inside cousins",
+        "schema": {
```

Comment:
There's another very similar example in a [comment](https://github.com/json-schema-org/JSON-Schema-Test-Suite/issues/359#issuecomment-624399662)

Reply:
The comment is what I used for `unevaluatedProperties.json`.

Reply:
We should have both tests for both keywords.

Reply:
I believe the "valid" cases are already there, that is, if you're referring to the "valid case" as the second test.

Reply:
If those cases are already in the suite, that's fine.